test/unit/test_inference.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
did we loose copyright for 2017? :P this should probably read: 
```
# Copyright 2017, 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Reply:
I assumed this should just reflect the latest year. Is it supposed to be a range?

Reply:
I think generally one should add each year in which changes were made to a file 
> To update the list of year numbers, add each year in which you have made nontrivial changes to the package
https://www.gnu.org/prep/maintain/html_node/Copyright-Notices.html